src/components/KebabMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { customersGetter, postCustomers } from 'src/api/CustomerApi';
+
+const CustomerTable = () => {
+  const queryClient = useQueryClient();
+  const customersQuery = useQuery('customers', customersGetter);
+
+  const deleteCustomerMutation = useMutation(postCustomers, {
```

Comment:
so the instructions kinda imply to use postCustomers, but really it needs to use a deleteCustomer api call that doesn't exist yet.    (I think its a mistake in the instructions).  You'll need add a new 'fake api call' here: https://github.com/content-services/React-typescript-demo/blob/main/src/api/CustomerApi.ts that does the opposite of adding a customer https://github.com/content-services/React-typescript-demo/blob/main/src/api/CustomerApi.ts#L27-L29 

Instead it should loop through the list of customers, removing the one that is to be deleted.

Reply:
mutators are fine to use here, but not strictly needed, it may just be a bit more complex  than other solutions.  But if you are comfortable with it, go for it!